### ISSUE TITLE
Feature request: Support `--enable-error-code` to allow for optional checks

### ISSUE THREAD

**Issue by RenDelaCruz** [ymyzk/mypy-playground#717]
It would be great to support mypy's `--enable-error-code` flag, to support some optional checks that don't have their own flag.  Error codes: https://mypy.readthedocs.io/en/stable/error_code_list2.html#check-that-expression-is-redundant-redundant-expr.

This includes very useful error codes like:
- `redundant-expr`
- `possibly-undefined`
- `truthy-bool`
- etc.

**Comment by sobolevn** [comment]
I agree, this is a critical missing feature: we can't really create repros for optional checks via this amazing tool.

I will create a PR.
